### PR TITLE
Namada spec changes contd.

### PR DESCRIPTION
**Pull request by cwgoes**
This PR does not finalise the content of all of the sub-specs, but it puts all the necessary pieces in place (I think).

**Instructions for reviewers of this PR**

Please render & read the specs locally. This can be done by running

```bash
cd documentation/spec; make serve
```

from the root of the repository. If you have any comments, please comment on the Markdown files directly in the "Files Changed" view on Github, or write a comment on this PR with any more general concerns or suggestions.

### CODE REVIEW

File: documentation/spec/src/SUMMARY.md [hunk 1]
```diff
@@ -23,8 +26,4 @@
         - [Reward distribution](./economics/proof-of-stake/reward-distribution.md)
     - [Shielded pool incentives](./economics/shielded-pool-incentives.md)
     - [Public goods funding](./economics/public-goods-funding.md)
-- [User interfaces](./user-interfaces.md)
-    - [Web wallet](./user-interfaces/web-wallet-interface.md)
-    - [Web explorer](./user-interfaces/web-explorer-interface.md)
-    - [External integrations](./user-interfaces/external-integrations.md)
-- [Further readhing](./further-reading.md)
\ No newline at end of file
+- [Further readhing](./further-reading.md)
```
Comment (tzemanovic) [inline documentation/spec/src/SUMMARY.md:29]:
```suggestion
- [Further reading](./further-reading.md)
```

File: documentation/spec/src/base-ledger/default-account.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## Default account
```
Comment (tzemanovic) [inline documentation/spec/src/base-ledger/default-account.md:1]: nit - top-level headings should be h1
```suggestion
# Default account
```

Comment (tzemanovic) [reply documentation/spec/src/base-ledger/default-account.md:1]: same in other files

**Comment by tzemanovic** [comment]
we also need to port stuff from https://github.com/anoma/anoma/tree/main/docs/src/specs

**Comment by gnosed** [comment]
There is also this `katex` bug that prevents some code blocks to display properly
```
/<span class="katex"><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mord mathnormal">G</span><span class="mord mathnormal">o</span><span class="mord mathnormal" style="margin-right:0.03588em;">v</span><span class="mord mathnormal" style="margin-right:0.02778em;">er</span><span class="mord mathnormal">nan</span><span class="mord mathnormal">ce</span><span class="mord mathnormal">A</span><span class="mord mathnormal">dd</span><span class="mord mathnormal">ress</span><span class="mord">/</span><span class="mord mathnormal">p</span><span class="mord mathnormal">ro</span><span class="mord mathnormal">p</span><span class="mord mathnormal">os</span><span class="mord mathnormal">a</span><span class="mord mathnormal" style="margin-right:0.01968em;">l</span><span class="mord">/</span></span></span></span>id/vote/<span class="katex"><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mord mathnormal">d</span><span class="mord mathnormal">e</span><span class="mord mathnormal" style="margin-right:0.01968em;">l</span><span class="mord mathnormal">e</span><span class="mord mathnormal" style="margin-right:0.03588em;">g</span><span class="mord mathnormal">a</span><span class="mord mathnormal">t</span><span class="mord mathnormal">i</span><span class="mord mathnormal">o</span><span class="mord"><span class="mord mathnormal">n</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.1514em;"><span style="top:-2.55em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mathnormal mtight">a</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord mathnormal">dd</span><span class="mord mathnormal">ress</span><span class="mord">/</span></span></span></span>voter_address: Enum(yay|nay)
```

**Comment by adrianbrink** [comment]
Can you apply these changes to the `specs` folder. I've renamed it in an earlier commit

**Pull request merged**